<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ONBRetrieveAvailableNodes</class>
 <widget class="QDialog" name="ONBRetrieveAvailableNodes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ONBRetrieveAvailableNodes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelIpSearch">
         <property name="text">
          <string>IP范围</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="IPLineEdit" name="lineEditIpStart"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="labelSymbol">
         <property name="text">
          <string>     ~      </string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="IPLineEdit" name="lineEditIpEnd"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelNodeTotal">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>节点总数:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelNodeTotalDisp">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelNodeIsConnected">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>可用节点:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelNodeIsConnectedDisp">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>IP</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备序号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>可链接状态</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonSearch">
       <property name="text">
        <string>筛选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="text">
        <string>复位</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IPLineEdit</class>
   <extends>QLineEdit</extends>
   <header>iplineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
